deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,382 @@
+# Copyright (C) 2024, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""Dashboard Saturation Tests Module
+
+This script allows you to run dashboard stress tests. These tests
+are performed with Artillery and Playwright. Artillery and
+Playwright must be installed for it to work properly.
+"""
+
+import argparse
+import json
+import pandas as pd
+import time
+import yaml
+
+from datetime import datetime
+from os import makedirs, getcwd
+from os.path import join, isabs
+from subprocess import run
+
+
+# Global Configuration Variables
+artillery_result_type = [
+    # Final statistics printed to the console at the end of a test
+    'aggregate',
+    # Statistics that print to the console while a test is running
+    'intermediate'
+]
+
+
+def check_artillery_result_types(types: list) -> None:
+    """Check that a valid type of Artillery result has been
+    chosen to generate the CSV.
+
+    Args:
+        types (list): List of types.
+    """
+    for value in types:
+        if value not in artillery_result_type:
+            msg = f'The Artillery result type to generate the CSV is not ' \
+                'valid. Accepted Values: {artillery_result_type}. ' \
+                'Value received {value}.'
+
+            raise Exception(msg)
+
+
+def format_directory(directory: str) -> str:
+    """Check the paths of the directories and format them if necessary.
+
+    Args:
+        args (str): Path of a directory.
+
+    Returns:
+        str: Formatted path.
+    """
+    return join(directory, '')
+
+
+def create_directories(directory: str) -> None:
+    """Create directory that does not exist.
+
+    Args:
+        directory (str): Path of directory.
+    """
+    makedirs(directory, exist_ok=True)
+
+
+def provide_directories(args: argparse) -> None:
+    """Format the paths of the directories and create them
```

Comment:
Follow https://google.github.io/styleguide/pyguide.html#382-modules. One line for the summary followed by a white line before a full description if needed